doc/sphinx-guides/source/_static/installation/files/var/www/dataverse/branding/custom-homepage-dynamic.html

```diff
@@ -94,10 +94,13 @@
                     <div class="search-widget input-group">
                         <input id="inputDataverseSearch" class="form-control" type="text" placeholder="" onkeydown="if (event.keyCode == 13) document.getElementById('btnDataverseSearch').click();">
                         <span class="input-group-btn">
-                            <button id="btnDataverseSearch" class="btn btn-default" type="button" onclick="window.location = '/dataverse/root?q=' + document.getElementById('inputDataverseSearch').value;return false;"><span class="glyphicon glyphicon-search"></span> Find</button>
+                            <button id="btnDataverseSearch" class="btn btn-default bootstrap-button-tooltip" type="button" data-original-title="Find" onclick="window.location = '/dataverse/root?q=' + document.getElementById('inputDataverseSearch').value;return false;"><span class="glyphicon glyphicon-search"></span></button>
```

Comment:
Is this a situation where we're not using something like:
'/dataverse/#{dataverseServiceBean.findRootDataverse().alias}?q='
because we don't have access to the app in the homepage?

Reply:
I'll let @mheppler confirm, but yeah this is html here and not xhtml. you could imagine if we needed to we could add an API to get the alias if the root. But seems like overkill here.

Reply:
Correct. Static custom html content has the root alias hardcoded all over the place. We do get plenty from the API already. Surprised we dont here or anywhere else already.